doc/release/upcoming_changes/17727.performance.rst

```diff
@@ -0,0 +1,7 @@
+Improved performance in integer division of NumPy arrays
+--------------------------------------------------------
+Integer division of NumPy arrays now uses libdivide when 

```

Comment:
We could add a link to libdivide, just to be nice :). But just an idea, it seems fine to me.

Reply:
Good idea yeah, I have linked them